legend-engine-xt-relationalStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/serialization/StreamingTempTableResultCSVSerializer.java

```diff
@@ -45,6 +45,7 @@ public class StreamingTempTableResultCSVSerializer extends CsvSerializer
 {
     private TempTableStreamingResult tempTableStreamingResult;
     private boolean withHeader;
```

Comment:
Is withHeader attribute required still?

Reply:
yes withHeader decides whether to print labels - csvPrinter.printRecord(columnLabels);